intezer_sdk/analysis.py

```diff
@@ -13,35 +14,38 @@
 
 class Analysis(object):
     def __init__(self,
-                 file_path=None,
-                 file_hash=None,
-                 dynamic_unpacking=None,
-                 api=None,
-                 static_unpacking=None):  # type: (str, str, bool, IntezerApi, bool) -> None
-        if (file_hash is not None) == (file_path is not None):
-            raise ValueError('Choose between file hash or file path analysis')
-
-        self.status = None  # type: consts.AnalysisStatusCode
-        self.analyses_id = None  # type: str
-        self._file_hash = file_hash  # type: str
-        self._dynamic_unpacking = dynamic_unpacking  # type: bool
-        self._static_unpacking = static_unpacking  # type: bool
-        self._file_path = file_path  # type: str
-        self._report = None  # type: dict
-        self._api = api or get_global_api()  # type: IntezerApi
-
-    def send(self, wait=False):  # type: (bool) -> None
-        if self.analyses_id:
+                 file_path: str = None,
+                 file_hash: str = None,
+                 file_stream: typing.BinaryIO = None,
+                 dynamic_unpacking: bool = None,
+                 api: IntezerApi = None,
```

Comment:
can you move it to be the last parameter

Reply:
Done